src/Symfony/Component/HttpClient/Response/CurlResponse.php

```diff
@@ -325,7 +335,7 @@ private static function select(ClientState $multi, float $timeout): int
             $timeout = min($timeout, 0.01);
         }
 
-        return curl_multi_select($multi->handle, $timeout);
+        return curl_multi_select($multi->handle[array_key_last($multi->handle)], $timeout);
```

Comment:
why the last key rather than the first one ?

Reply:
Because the last handle is the only one for which we're sure there is pending activity (since handles past the `0` are all removed when they're empty).